### PR TITLE
Chrome/Firefox

### PR DESCRIPTION
**Pull request by pwright**
https://issues.jboss.org/browse/INTLY-3566

### CODE REVIEW

**Comment by pwright** [comment]
@briandooley mind taking a look

File: walkthroughs/1-ionic-showcase/walkthrough.adoc [hunk 1]
```diff
@@ -417,24 +418,24 @@ Verify that you followed each step in the procedure above.  If you are still hav
 
 === Demonstrationg offline support
 
-. On your mobile device:
+. Using the Firefox browser:
 .. Log into the showcase app.
-.. Activate airplane mode or disable network connectivity.
+.. Simulate offline mode. In Firefox, select *Work Offline* from the *Web Developers* menu.
 .. Create a new task.
```
Comment (jhellar) [inline walkthroughs/1-ionic-showcase/walkthrough.adoc:424]: Tried to follow the steps in [1] - when I go offline and try to create new task, page for creating new task does not appear.

[1] https://routez5ynktzk-codeready.apps.dbizzarr-b9fe.open.redhat.com/tasks

File: walkthroughs/1-ionic-showcase/walkthrough.adoc [hunk 2]
```diff
@@ -384,23 +385,23 @@ Verify that you followed each step in the procedure above.  If you are still hav
...
 .. Create a new task using *+* icon.
 .. Enter some task text  and click *Create*.
 
-. On your mobile device:
+. Using the Firefox browser:
 .. Check that the same task appears in the *Manage Tasks* page.
 .. Make some changes to the task.
 
-. On your laptop:
-.. Check that the task changes are appear.
```
Comment (briandooley) [inline walkthroughs/1-ionic-showcase/walkthrough.adoc:403]: Are we saying that the user has to have chrome and firefox installed on their mobile device?  If 2 different browsers are required, can we just leave the choice of them up to the user?\

Comment (pwright) [reply walkthroughs/1-ionic-showcase/walkthrough.adoc:403]: two browser from laptop/desktop is the only config supported.

**Comment by pwright** [comment]
do not merge if https://github.com/aerogear/ionic-showcase/pull/254 is merged

**Comment by pwright** [comment]
closing in favor of https://github.com/aerogear/ionic-showcase/pull/254

**Pull request closed without merging**